sensing/pointcloud_preprocessor/schema/distortion_corrector.schema.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Remove this file as it already exists.

Reply:
Sorry, I only found one file here, will keep it currently.

Reply:
https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/pointcloud_preprocessor/schema/distortion_corrector_node.schema.json

Reply:
Oh! I see, got it. Will rebase my branch. Thank you!